src/Core/RulesConfigurableEventHandlerInterface.php

```diff
@@ -90,6 +90,7 @@ public function extractConfigurationFormValues(array &$form, FormStateInterface
   public function validate();
 
   /**
+   * todo
```

Comment:
I replaced getEventNameSuffix with getEventNamePrefix. Is it ok?

If yes, I will change the comment